.github/workflows/continuous-integration.yml

```diff
@@ -468,6 +469,10 @@ jobs:
         id: tests
         run: echo ::set-output name=selected::$TESTS
 
+      - name: Set output of IS_MASTER_BUILD
```

Comment:
If you are already setting it as a global variable for `cypress-test-prep`, why would you need to reset it as an output? Maybe I am missing something? But you should be able to get rid of this step and just have `env.IS_MASTER_BUILD` in output

Reply:
Ohh...

https://docs.github.com/en/actions/reference/environment-variables#about-environment-variables

> You can define environment variables for a step, job, or entire workflow using the jobs.<job_id>.steps[*].env, jobs.<job_id>.env, and env keywords.

Reply:
Oooh, I didn't know you can access it from `needs.<job>.<job_id>.env`. TIL!!!!

I was saying in the output you could have done 

```
outputs:
  is_master_build: $ {{ env.is_master_build }}
```

But that is a pretty neat trick! Thanks for sharing that doc

Reply:
I tried the following but couldn't reference that env var:
- `needs.cypress-tests-prep.env.IS_MASTER_BUILD`
- `jobs.cypress-tests-prep.env.IS_MASTER_BUILD`
- `github.workflow.jobs.cypress-tests-prep.env.IS_MASTER_BUIL`

So changed it to this, like you suggested:
```
outputs:
  is_master_build: $ {{ env.IS_MASTER_BUILD }}
```